CHANGELOG.md

```diff
@@ -11,6 +11,7 @@
 
 - Mark `HttpServer::new_from_fd` as `unsafe` as the correctness of the unsafe code
   in this method relies on an invariant the caller has to uphold
+- Set `Content-Length: 0` when setting an empty body in the response.
```

Comment:
Mention that this is in MMDS, or does it also apply to the HTTP API? If yes, mention both.

Reply:
This change is not specific to MMDS, generic. micro-http is a separate crate from Firecracker, so I think we should not mention MMDS.

Reply:
mentioned that the client may wait for the connection closed or timeout.